src/analytics/ledger/active_addresses.rs

```diff
@@ -5,29 +5,26 @@ use std::collections::HashSet;
 
 use time::{Duration, OffsetDateTime};
 
-use super::TransactionAnalytics;
-use crate::types::{
-    ledger::{LedgerOutput, LedgerSpent, MilestoneIndexTimestamp},
-    stardust::block::Address,
-};
+use super::*;
+use crate::types::stardust::block::Address;
 
-pub struct AddressActivityMeasurement {
-    pub count: usize,
+pub(crate) struct AddressActivityMeasurement {
```

Comment:
Could we even use `pub(super)`?

Reply:
It turns out we cannot, as `pub(super)` types cannot be re-exported.